src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,137 @@
+
+**compute**
+
+* Add simulate-eviction operation for VM and VMSS (#14133)
+
+**CosmosDB**
+
+* Recent features: Autoscale, IpRules, EnableFreeTier and EnableAnalyticalStorage (#13985)
+
+**EventGrid**
+
+* Add CLI support for 2020-04-01-preview and mark preview features with is_Preview=True (#14027)
+
+**Find**
+
+* Fix #14094 az find Fix Queries failing when not logged in and when telemetry is disabled (#14243)
+
+**HDInsight**
+
+* Add two commands to support hdinsight node reboot feature (#14005)
+
+**Monitor**
+
+* Remove preview flag for commands under Log Analytics workspace (#14064)
+* `az monitor diagnostic-settings subscription`: Support diagnositc settings for subscription (#14157)
+* `az monitor metrics`: support ',' and '|' in metric name (#14254)
+* `az monitor log-analytics workspace data-export`: support log analytics data export (#14155)
+
+**Network**
+
+* `az network application-gateway frontend-ip update`: Deprecating the --public-ip-address parameter (#13891)
+* Bump azure-mgmt-network to 11.0.0 (#13957)
+* `az network express-route gateway connection`: support routing configuration (#14256)
+* `az network virtual-appliance`: Support Azure network virtual appliance. (#14224)
+* Application Gateway support private link feature (#14185)
+
+**PolicyInsights**
+
+* `az policy state`: add trigger-scan command to trigger policy compliance evaluations (#12910)
+* `az policy state list`: expose versions of policy entities in each compliance record (#12910)
+
+**Profile**
+
+* `az account get-access-token`: Show expiresOn for Managed Identity (#14128)
+
+**Rdbms**
+
+* Add Infrastructure Encryption for Azure Postgres and MySQL (#14097)
+
+**RDBMS**
+
+* Support Minimum TLS version (#14166)
+
+**security**
+
+* Add regulatory compliance CLI (#14103)
+
+**Security**
+
+* Add allowed_connections commands (#14190)
+* Add Adaptive network hardeningss commands (#14260)
+* Add adaptive_application_controls commands (#14278)
+
+**SignalR**
+
+* Add features including managing private endpoint connections, network rules and upstream (#14008)
+
+**SQL**
+
+* `az sql mi create`, `az sql mi update`: Add `--tags` parameter to support resource tagging (#13479)
+* `az sql mi failover`: Support failover from primary or secondary point (#14242)
+
+**Storage**
+
+* `az storage account create/update`: Add --allow-blob-public-access to allow or disallow public access for blob and containers (#13986)
+* `az storage account create/update`: Add `--min-tls-version` to support setting the minimum TLS version to be permitted on requests to storage. (#14131)
+* Remove check in token credential (#14134)
+* Fix the storage account name in examples (#14062)
+
+**Thoth**
+
+* : recommend_recovery_options: version: "2.8.0", command: "account", parameters: "['--debug']", extension: "None" (#14180)
+* : User must agree to telemetry before failure recovery recommendations can be presented. (#14180)
```

Comment:
@fengzhou-msft. This is caused by an incorrect history extraction. https://github.com/Azure/azure-cli/pull/14180 has these lines as code:

```
[Thoth]: recommend_recovery_options: version: "2.8.0", command: "account", parameters: "['--debug']", extension: "None"
[Thoth]: User must agree to telemetry before failure recovery recommendations can be presented.
```